### PR TITLE
kni: numaresources: enable security scanning

### PR DESCRIPTION
**Pull request by ffromani**
per https://docs.ci.openshift.org/docs/how-tos/add-security-scanning/

### CODE REVIEW

File: ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml [hunk 1]
```diff
@@ -130,6 +130,13 @@ tests:
           cpu: 500m
           memory: 1Gi
     workflow: optional-operators-ci-aws
+- as: security
+  optional: true
```
Comment (shajmakh) [inline ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:134]: I'm just pumping to ensure that setting this to optional PRs that didn't pass the check won't be a blocker for the merge. that is what we want right?

Comment (ffromani) [reply ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:134]: this is totally the intention, yes. The lane should be informational, not blocking.

**Comment by ffromani** [comment]
/pj-rehearse pull-ci-openshift-kni-numaresources-operator-main-security

**Comment by ffromani** [comment]
the lane is optional, and failures are expected

**Comment by ffromani** [comment]
once this is merged and stabilizes, we will need to enable also on 4.14 and 4.12

**Comment by ffromani** [comment]
is the bot extremely slow or it's we can't merge PRs with failed jobs - even if they are optional?

**Comment by ffromani** [comment]
> is the bot extremely slow or it's we can't merge PRs with failed jobs - even if they are optional?

or both?

**Pull request merged**